app/enterprise/1.3-x/deployment/installation/docker.md

```diff
@@ -1,172 +1,153 @@
+```
+
+Note: the license data must contain only straight quotes to be considered valid JSON. (`'` and `"`, not `’` or `“`)
+
+## Step 5. Prepare the Kong Database
+
+```
+docker run --rm --network=kong-ee-net \
+  -e "KONG_DATABASE=postgres" \
+  -e "KONG_PG_HOST=kong-ee-database" \
+  -e "KONG_LICENSE_DATA=$KONG_LICENSE_DATA" \
+  kong-ee kong migrations bootstrap
+```
+
+## Step 6. Start Kong with Kong Manager and Developer Portal Enabled
+
+```
+docker run -d --name kong-ee --network=kong-ee-net \
+   -e "KONG_DATABASE=postgres" \
+   -e "KONG_PG_HOST=kong-ee-database" \
+   -e "KONG_PROXY_ACCESS_LOG=/dev/stdout" \
+   -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout" \
+   -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
+   -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
+   -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
+   -e "KONG_PORTAL=on" \
+   -e "KONG_LICENSE_DATA=$KONG_LICENSE_DATA" \
+   -e "KONG_PASSWORD=<SOMETHING-YOU-KNOW>" \
+   -e "KONG_PORTAL_GUI_HOST=<DNSorIP>:8003" \
+   -e "KONG_ADMIN_GUI_URL=http://<DNSorIP>:8002" \
+   -p 8000:8000 \
+   -p 8443:8443 \
+   -p 8001:8001 \
+   -p 8444:8444 \
+   -p 8002:8002 \
+   -p 8445:8445 \
+   -p 8003:8003 \
+   -p 8004:8004 \
+   kong-ee
+```
+  
+**Notes** 
+- For `KONG_PASSWORD`, replace <SOMETHING-YOU-KNOW> with a valid password that only you know.
+- For `KONG_PORAL_GUI_HOST` and `KONG_ADMIN_GUI_URL`, replace `<DNSorIP>` with with the DNS name or IP of the Docker host.
+   * The DNS or IP address for `KONG_PORAL_GUI_HOST` should _not_ be preceded with a protocol, e.g. `http://`.
+   * `KONG_ADMIN_GUI_URL` _should_ have a protocol, e.g., `http://`. 
+  
+**Docker on Windows users:** Instead of the `KONG_LICENSE_DATA` environment variable, use the [volume bind](https://docs.docker.com/engine/reference/commandline/run/#options) option. For example, assuming you've saved your `license.json` file into `C:\temp`, use `--volume /c/temp/license.json:/etc/kong/license.json` to specify the license file.
+
+## Step 7. Finalize Configuration and Verify Success of Kong Installation
+
+```bash
+$ curl -i -X GET --url http://localhost:8001/services
+```
+
+You should receive an `HTTP/1.1 200 OK` message.
+      
+Verify Kong Manager is running by accessing it via the URL specified in KONG_ADMIN_GUI_URL in [Step 6](#step-6-start-kong-with-kong-manager-and-developer-portal-enabled).
```

Comment:
In a future iteration, let's define links at the bottom and use variables instead. That way if the name of the ID changes, we only need to update the link in one place.

```suggestion
Verify Kong Manager is running by accessing it via the URL specified in KONG_ADMIN_GUI_URL in [Step 6](#step-6-start-kong-enterprise-with-kong-manager-and-kong-developer-portal-enabled).
```